package-lock.json

```diff
@@ -434,6 +445,17 @@
             "lodash": "^4.17.13",
             "to-fast-properties": "^2.0.0"
           }
+        },
+        "chalk": {
```

Comment:
Ok I'm confused by this 3x `chalk` entries 🤔

Reply:
Package.lock files are organized automatically. Don't worry about that.